src/App.css

```diff
@@ -28,6 +28,49 @@
   color: #61dafb;
 }
 
+.my-task {
+  padding-left: 50px;
+  width: 250 px;
+  height: 50px;
+  border: solid;
+  display: flex;
+  justify-content: center;
+}
+
+.row {
+  display: table;
+  width: 800px;
+  padding: 10px;
+}
```

Comment:
`display: table` is outdated since it's very hard to use in a complex layout. I would suggest using `display: flex` like how you have it in `.my-task`

[Here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is a visual cheat sheet that you can use to learn more about how it work